_posts/2020-04-01-newsletter.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Origami Newsletter, March 2020
+description: This issue features remote first working, deep spring cleaning, and an improved release workflow.
+author: Lee Moody
+tags:
+  - Newsletter
+---
+
+<abbr title="Too long; didn't read">
+	<strong>
+	TL;DR:
+	</strong>
+</abbr> {{page.description}}
+
+## Top Things
+
+These are some of the bigger things we've done over the last month.
+
+### Remote First
+
+It's hard to believe we were largely working from an office at the beginning of this month. A reminder that we're here to help. Our level of support is the same, and we’re happy to help via a hangout or Slack any time. 😊
```

Comment:
great :)

Reply:
_any time_, should we remove that? We are happy to help when we are working, whatever those hours are 😅

Reply:
that’s a good point.  this might be a good place to mention Office Hours too

Reply:
Good point! 😅 I'll just remove that. Office hours are mentioned below 😊 

```suggestion
It's hard to believe we were largely working from an office at the beginning of this month. A reminder that we're here to help. Our level of support is the same, and we’re happy to help via a hangout or Slack. 😊
```